src/Service.Tests/SqlTests/GraphQLSupportedTypesTests/GraphQLSupportedTypesTestsBase.cs

```diff
@@ -171,40 +133,66 @@ public async Task QueryTypeColumnFilterAndOrderBy(string type, string filterOper
+        [DataRow(DATETIME_TYPE, "lt", "'1753-01-01'", "\"1753-01-01\"", " < ",
+            DisplayName = "datetime type filter and orderby test with lt operator and specific value '1753-01-01'.")]
+        [DataRow(DATETIME_TYPE, "lte", "'1753-01-01'", "\"1753-01-01\"", " <= ",
+            DisplayName = "datetime type filter and orderby test with lte operator and specific value '1753-01-01'.")]
+        [DataRow(DATETIME_TYPE, "neq", "'1753-01-01'", "\"1753-01-01\"", "!=",
+            DisplayName = "datetime type filter and orderby test with neq operator and specific value '1753-01-01'.")]
+        [DataRow(DATETIME_TYPE, "eq", "'1753-01-01'", "\"1753-01-01\"", "=",
+            DisplayName = "datetime type filter and orderby test with eq operator and specific value '1753-01-01'.")]
+        [DataRow(DATETIME_TYPE, "gt", "'1999-01-08 10:23:00'", "\"1999-01-08 10:23:00\"", " > ",
+            DisplayName = "datetime type filter and orderby test with gt operator and specific value '1999-01-08 10:23:00'.")]
+        [DataRow(DATETIME_TYPE, "gte", "'1999-01-08 10:23:00'", "\"1999-01-08 10:23:00\"", " >= ",
+            DisplayName = "datetime type filter and orderby test with gte operator and specific value '1999-01-08 10:23:00'.")]
+        [DataRow(DATETIME_TYPE, "lt", "'9999-12-31 23:59:59'", "\"9999-12-31 23:59:59\"", " < ",
+            DisplayName = "datetime type filter and orderby test with lt operator and specific value '9999-12-31 23:59:59'.")]
+        [DataRow(DATETIME_TYPE, "lte", "'9999-12-31 23:59:59'", "\"9999-12-31 23:59:59\"", " <= ",
+            DisplayName = "datetime type filter and orderby test with lte operator and specific value '9999-12-31 23:59:59'.")]
+        [DataRow(DATETIME_TYPE, "neq", "'1999-01-08 10:23:00'", "\"1999-01-08 10:23:00\"", "!=",
+            DisplayName = "datetime type filter and orderby test with neq operator and specific value '1999-01-08 10:23:00'.")]
+        [DataRow(DATETIME_TYPE, "eq", "'1999-01-08 10:23:00'", "\"1999-01-08 10:23:00\"", "=",
+            DisplayName = "datetime type filter and orderby test with eq operator and specific value '1999-01-08 10:23:00'.")]
+        [DataRow(DATETIME_TYPE, "lte", "'2079-06-06'", "\"2079-06-06\"", " <= ",
+            DisplayName = "datetime type filter and orderby test with lte operator and specific value '2079-06-06'.")]
+        [DataRow(DATETIME_TYPE, "neq", "'1999-01-08 10:23:54'", "\"1999-01-08 10:23:54\"", "!=",
+            DisplayName = "datetime type filter and orderby test with neq operator and specific value '1999-01-08 10:23:54'.")]
+        [DataRow(DATETIMEOFFSET_TYPE, "neq", "'1999-01-08 10:23:54.9999999-14:00'", "\"1999-01-08 10:23:54.9999999-14:00\"", "!=",
+            DisplayName = "datetimeoffset type filter and orderby test with neq operator")]
+        [DataRow(DATETIMEOFFSET_TYPE, "eq", "'1999-01-08 10:23:54.9999999-14:00'", "\"1999-01-08 10:23:54.9999999-14:00\"", "=",
```

Comment:
Is there a test testing max value for DATETIMEOFFSET_TYPE?

Reply:
added now